UI/window-projector.cpp

```diff
@@ -35,8 +39,11 @@ OBSProjector::OBSProjector(QWidget *widget, obs_source_t *source_, int monitor,
 	windowHandle()->setProperty("isOBSProjectorWindow", true);
 
 #if defined(__linux__) || defined(__FreeBSD__) || defined(__DragonFly__)
-	// Prevents resizing of projector windows
-	setAttribute(Qt::WA_PaintOnScreen, false);
+	if (obs_get_nix_platform() == OBS_NIX_PLATFORM_X11_EGL)
```

Comment:
Prefer asking Qt's backend than libobs for this case.

```suggestion
	if (!QApplication::platformName().contains("wayland"))
```